sdcm/utils/decorators.py

```diff
@@ -155,6 +158,96 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+class LatencyCalculatorMixedResult(GenericResultTable):
```

Comment:
I think we should put those in it's own module

Reply:
like for all results gathered in argus or just for performance?

Reply:
I don't know yet, but those are not decorators, as the module says.

I think it makes sense to gather all of those in one place, so it would be one place to checkout what are we collecting.

even that no I see lots of the things names/descriptions are created dynamically, and it would be hard to see, with opening up the DB (or have argus to show all of them)

Reply:
Moved to different module `argus_results.py`